teps/0039-add-variable-retries-and-retrycount.md

```diff
@@ -0,0 +1,107 @@
+---
+status: proposed
+title: Add Variable `retries` and `retry-count`
+creation-date: '2020-10-14'
+last-updated: '2020-12-17'
+authors:
+- '@yaoxiaoqi'
+---
+
+# TEP-0039: Add Variable `retries` and `retry-count`
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+  - [Notes/Constraints/Caveats](#notesconstraintscaveats)
```

Comment:
I think these two sub-sections (user stories & notes/constraints) have been removed so can be deleted here too.